### DISCUSSION TITLE
Need a way to query links or link if there are thousands of data

### DISCUSSION THREAD

**Discussion by dgandhi17** [isar/isar#61] [Help]
Can you provide a way where I want to filter on links instead of property. As per document links are lazy and we have to call load on that so not sure how a query will look like if we need to query based on links or link.

for ex.

class ModelA{
 string property1;
 IsarLink model = IsarLink();
}

isar.modalas.where().filter().mode ?

@leisim  can you provide a way as the documentation is out dated.

**Comment by simc** [comment]
@dgandhi17 I'll update the docs with the next version :+1:
